src/gui/tray/Window.qml

```diff
@@ -139,6 +147,90 @@ ApplicationWindow {
         }
     }
 
+    Drawer {
+        id: fileDetailsDrawer
+        width: parent.width
+        height: parent.height
+        padding: 0
+        edge: Qt.RightEdge
+        modal: false
+        visible: false
+
+        background: Rectangle {
+            radius: Systray.useNormalWindow ? 0.0 : Style.trayWindowRadius
+            border.width: Style.trayWindowBorderWidth
+            border.color: Style.menuBorder
+            color: Style.backgroundColor
+        }
+
+        property var folderAccountState: ({})
+        property string fileLocalPath: ""
+        property int pageToShow: 0
+
+        function openFileDetails(accountState, localPath, fileDetailsPage) {
+            console.log(`About to show file details view in tray for ${localPath}`);

```

Comment:
should you not check that the drawer is really closed in case you would trigger it twice inadvertently ?

Reply:
Added a check now